pkg/types/types.go

```diff
@@ -329,32 +264,33 @@ func (m *ComMatrix) ToNFTables() ([]byte, error) {
 	return []byte(result), nil
 }
 
-// SortAndRemoveDuplicates removes duplicates in the matrix and sort it.
-func (m *ComMatrix) SortAndRemoveDuplicates() {
+// SortAndRemoveDuplicates sorts ComDetails and removes duplicates.
+func SortAndRemoveDuplicates(cds []ComDetails) []ComDetails {
```

Comment:
why u changed this (m *ComMatrix) SortAndRemoveDuplicates()?

Reply:
To use it in the new matrix diff package

Reply:
yah but u can create on the diff as ComMatrix and not []ComDetails